src/Uno.Extensions.Templates/content/unoapp-extensions/Directory.Build.nocpm.props

```diff
@@ -9,9 +9,18 @@
 		<SynthesizeLinkMetadata>true</SynthesizeLinkMetadata>
 
 		<TreatWarningsAsErrors>True</TreatWarningsAsErrors>
-	  	<NoWarn>$(NoWarn);Uno0001;CS1998;CA1416;NU1507</NoWarn>
+		<NoWarn>$(NoWarn);Uno0001;CS1998;CA1416;NU1507</NoWarn>
 	</PropertyGroup>
 
+	<ItemGroup Condition=" $(UnoSingleLocationProject) == 'true' ">
```

Comment:
This feels sloppy - is this consistent with the Uno vanilla template, or is this specific to the extensions template